#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_use_the_entities_from_sgml_data.dpatch by Adam Di Carlo <aph@debian.org>
##     and Daniel Leidert (dale) <daniel.leidert@wgdd.de>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.x/dbcentx.mod: don't install and reference ISO XML
## DP:   entities, which are in the sgml-data package
## DP:
## DP: * docbook-4.x/{docbook.cat,catalog.xml}: Removes the ISO XML entity
## DP:   references, which are part of sgml-data
## DP:
## DP: <URL:http://bugs.debian.org/99993>
## DP: <URL:http://bugs.debian.org/102605>
## DP: <URL:http://bugs.debian.org/232600>
## DP: <URL:http://bugs.debian.org/305736>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion3.1.5.1-1>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.1.2-5>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.1.2-7>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-5>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-11>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>
## DP:
## DP: for i in `find . -type f` ; \
## DP: 		do sed -e 's/^\"ent\/iso-/\"\/usr\/share\/xml\/entities\/xml-iso-entities-8879\.1986\/ISO/g' < $i > $i.new ; \
## DP: 		mv $i.new $i ; \
## DP: done

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.0/dbcentx.mod docbook-xml-4.5/docbook-4.0/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.0/dbcentx.mod	2000-05-18 21:40:49.000000000 +0200
+++ docbook-xml-4.5/docbook-4.0/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -52,7 +52,7 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/iso-amsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 %ISOamsa;
 <!--end of ISOamsa.module-->]]>
 
@@ -60,7 +60,7 @@
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/iso-amsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 %ISOamsb;
 <!--end of ISOamsb.module-->]]>
 
@@ -68,7 +68,7 @@
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/iso-amsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 %ISOamsc;
 <!--end of ISOamsc.module-->]]>
 
@@ -76,7 +76,7 @@
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/iso-amsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 %ISOamsn;
 <!--end of ISOamsn.module-->]]>
 
@@ -84,7 +84,7 @@
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/iso-amso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 %ISOamso;
 <!--end of ISOamso.module-->]]>
 
@@ -92,7 +92,7 @@
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/iso-amsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 %ISOamsr;
 <!--end of ISOamsr.module-->]]>
 
@@ -100,7 +100,7 @@
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/iso-box.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 %ISObox;
 <!--end of ISObox.module-->]]>
 
@@ -108,7 +108,7 @@
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/iso-cyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 %ISOcyr1;
 <!--end of ISOcyr1.module-->]]>
 
@@ -116,7 +116,7 @@
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/iso-cyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 %ISOcyr2;
 <!--end of ISOcyr2.module-->]]>
 
@@ -124,7 +124,7 @@
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/iso-dia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 %ISOdia;
 <!--end of ISOdia.module-->]]>
 
@@ -132,7 +132,7 @@
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/iso-grk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 %ISOgrk1;
 <!--end of ISOgrk1.module-->]]>
 
@@ -140,7 +140,7 @@
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/iso-grk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 %ISOgrk2;
 <!--end of ISOgrk2.module-->]]>
 
@@ -148,7 +148,7 @@
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/iso-grk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 %ISOgrk3;
 <!--end of ISOgrk3.module-->]]>
 
@@ -156,7 +156,7 @@
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/iso-grk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 %ISOgrk4;
 <!--end of ISOgrk4.module-->]]>
 
@@ -164,7 +164,7 @@
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/iso-lat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 %ISOlat1;
 <!--end of ISOlat1.module-->]]>
 
@@ -172,7 +172,7 @@
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/iso-lat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 %ISOlat2;
 <!--end of ISOlat2.module-->]]>
 
@@ -180,7 +180,7 @@
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/iso-num.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 %ISOnum;
 <!--end of ISOnum.module-->]]>
 
@@ -188,7 +188,7 @@
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/iso-pub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 %ISOpub;
 <!--end of ISOpub.module-->]]>
 
@@ -196,7 +196,7 @@
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/iso-tech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 %ISOtech;
 <!--end of ISOtech.module-->]]>
 
diff -urNad docbook-xml-4.5~/docbook-4.1.2/dbcentx.mod docbook-xml-4.5/docbook-4.1.2/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.1.2/dbcentx.mod	2000-08-27 17:19:42.000000000 +0200
+++ docbook-xml-4.5/docbook-4.1.2/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -52,7 +52,7 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/iso-amsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 %ISOamsa;
 <!--end of ISOamsa.module-->]]>
 
@@ -60,7 +60,7 @@
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/iso-amsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 %ISOamsb;
 <!--end of ISOamsb.module-->]]>
 
@@ -68,7 +68,7 @@
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/iso-amsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 %ISOamsc;
 <!--end of ISOamsc.module-->]]>
 
@@ -76,7 +76,7 @@
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/iso-amsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 %ISOamsn;
 <!--end of ISOamsn.module-->]]>
 
@@ -84,7 +84,7 @@
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/iso-amso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 %ISOamso;
 <!--end of ISOamso.module-->]]>
 
@@ -92,7 +92,7 @@
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/iso-amsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 %ISOamsr;
 <!--end of ISOamsr.module-->]]>
 
@@ -100,7 +100,7 @@
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/iso-box.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 %ISObox;
 <!--end of ISObox.module-->]]>
 
@@ -108,7 +108,7 @@
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/iso-cyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 %ISOcyr1;
 <!--end of ISOcyr1.module-->]]>
 
@@ -116,7 +116,7 @@
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/iso-cyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 %ISOcyr2;
 <!--end of ISOcyr2.module-->]]>
 
@@ -124,7 +124,7 @@
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/iso-dia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 %ISOdia;
 <!--end of ISOdia.module-->]]>
 
@@ -132,7 +132,7 @@
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/iso-grk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 %ISOgrk1;
 <!--end of ISOgrk1.module-->]]>
 
@@ -140,7 +140,7 @@
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/iso-grk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 %ISOgrk2;
 <!--end of ISOgrk2.module-->]]>
 
@@ -148,7 +148,7 @@
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/iso-grk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 %ISOgrk3;
 <!--end of ISOgrk3.module-->]]>
 
@@ -156,7 +156,7 @@
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/iso-grk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 %ISOgrk4;
 <!--end of ISOgrk4.module-->]]>
 
@@ -164,7 +164,7 @@
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/iso-lat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 %ISOlat1;
 <!--end of ISOlat1.module-->]]>
 
@@ -172,7 +172,7 @@
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/iso-lat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 %ISOlat2;
 <!--end of ISOlat2.module-->]]>
 
@@ -180,7 +180,7 @@
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/iso-num.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 %ISOnum;
 <!--end of ISOnum.module-->]]>
 
@@ -188,7 +188,7 @@
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/iso-pub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 %ISOpub;
 <!--end of ISOpub.module-->]]>
 
@@ -196,7 +196,7 @@
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/iso-tech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 %ISOtech;
 <!--end of ISOtech.module-->]]>
 
diff -urNad docbook-xml-4.5~/docbook-4.1.2/docbook.cat docbook-xml-4.5/docbook-4.1.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.1.2/docbook.cat	2000-08-27 17:19:42.000000000 +0200
+++ docbook-xml-4.5/docbook-4.1.2/docbook.cat	2007-03-25 23:42:57.000000000 +0200
@@ -32,28 +32,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook XML Notations V4.1.2//EN" "dbnotnx.mod"
 PUBLIC "-//OASIS//ENTITIES DocBook XML Character Entities V4.1.2//EN" "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "ent/iso-dia.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "ent/iso-num.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "ent/iso-pub.ent"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "ent/iso-tech.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "ent/iso-lat1.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "ent/iso-lat2.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "ent/iso-grk1.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "ent/iso-grk2.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "ent/iso-grk3.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "ent/iso-grk4.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "ent/iso-amsa.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "ent/iso-amsb.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "ent/iso-amsc.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "ent/iso-amsn.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "ent/iso-amso.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "ent/iso-amsr.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "ent/iso-box.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "ent/iso-cyr1.ent"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "ent/iso-cyr2.ent"
-
   -- End of catalog data for DocBook XML V4.1.2 ............................. --
   -- ...................................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.2/catalog.xml docbook-xml-4.5/docbook-4.2/catalog.xml
--- docbook-xml-4.5~/docbook-4.2/catalog.xml	2002-07-17 18:14:41.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/catalog.xml	2007-03-25 23:42:57.000000000 +0200
@@ -49,66 +49,6 @@
 <public publicId="-//OASIS//ENTITIES DocBook Character Entities V4.2//EN"
         uri="dbcentx.mod"/>
 
-<!-- ...................................................................... -->
-<!-- ISO entity sets ...................................................... -->
-
-<public publicId="ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-        uri="ent/iso-dia.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-        uri="ent/iso-num.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Publishing//EN//XML"
-        uri="ent/iso-pub.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES General Technical//EN//XML"
-        uri="ent/iso-tech.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-        uri="ent/iso-lat1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-        uri="ent/iso-lat2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-        uri="ent/iso-grk1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-        uri="ent/iso-grk2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-        uri="ent/iso-grk3.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-        uri="ent/iso-grk4.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-        uri="ent/iso-amsa.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-        uri="ent/iso-amsb.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-        uri="ent/iso-amsc.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-        uri="ent/iso-amsn.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-        uri="ent/iso-amso.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-        uri="ent/iso-amsr.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-        uri="ent/iso-box.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-        uri="ent/iso-cyr1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-        uri="ent/iso-cyr2.ent"/>
-
 <!-- End of catalog data for DocBook XML V4.2 ............................. -->
 <!-- ...................................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.2/dbcentx.mod docbook-xml-4.5/docbook-4.2/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.2/dbcentx.mod	2002-07-17 18:14:41.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -173,133 +173,133 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/iso-amsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 <!--end of ISOamsa.module-->]]>
 
 <!ENTITY % ISOamsb.module "INCLUDE">
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/iso-amsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 <!--end of ISOamsb.module-->]]>
 
 <!ENTITY % ISOamsc.module "INCLUDE">
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/iso-amsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 <!--end of ISOamsc.module-->]]>
 
 <!ENTITY % ISOamsn.module "INCLUDE">
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/iso-amsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 <!--end of ISOamsn.module-->]]>
 
 <!ENTITY % ISOamso.module "INCLUDE">
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/iso-amso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 <!--end of ISOamso.module-->]]>
 
 <!ENTITY % ISOamsr.module "INCLUDE">
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/iso-amsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 <!--end of ISOamsr.module-->]]>
 
 <!ENTITY % ISObox.module "INCLUDE">
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/iso-box.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 <!--end of ISObox.module-->]]>
 
 <!ENTITY % ISOcyr1.module "INCLUDE">
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/iso-cyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 <!--end of ISOcyr1.module-->]]>
 
 <!ENTITY % ISOcyr2.module "INCLUDE">
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/iso-cyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 <!--end of ISOcyr2.module-->]]>
 
 <!ENTITY % ISOdia.module "INCLUDE">
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/iso-dia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 <!--end of ISOdia.module-->]]>
 
 <!ENTITY % ISOgrk1.module "INCLUDE">
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/iso-grk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 <!--end of ISOgrk1.module-->]]>
 
 <!ENTITY % ISOgrk2.module "INCLUDE">
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/iso-grk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 <!--end of ISOgrk2.module-->]]>
 
 <!ENTITY % ISOgrk3.module "INCLUDE">
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/iso-grk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 <!--end of ISOgrk3.module-->]]>
 
 <!ENTITY % ISOgrk4.module "INCLUDE">
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/iso-grk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 <!--end of ISOgrk4.module-->]]>
 
 <!ENTITY % ISOlat1.module "INCLUDE">
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/iso-lat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 <!--end of ISOlat1.module-->]]>
 
 <!ENTITY % ISOlat2.module "INCLUDE">
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/iso-lat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 <!--end of ISOlat2.module-->]]>
 
 <!ENTITY % ISOnum.module "INCLUDE">
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/iso-num.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 <!--end of ISOnum.module-->]]>
 
 <!ENTITY % ISOpub.module "INCLUDE">
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/iso-pub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 <!--end of ISOpub.module-->]]>
 
 <!ENTITY % ISOtech.module "INCLUDE">
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/iso-tech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 <!--end of ISOtech.module-->]]>
 
 <!--end of xml.features-->]]>
diff -urNad docbook-xml-4.5~/docbook-4.2/docbook.cat docbook-xml-4.5/docbook-4.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.2/docbook.cat	2002-07-17 18:14:41.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/docbook.cat	2007-03-25 23:42:57.000000000 +0200
@@ -46,65 +46,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.2//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-       "ent/iso-dia.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-       "ent/iso-num.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-       "ent/iso-pub.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-       "ent/iso-tech.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-       "ent/iso-lat1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-       "ent/iso-lat2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-       "ent/iso-grk1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-       "ent/iso-grk2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-       "ent/iso-grk3.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-       "ent/iso-grk4.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-       "ent/iso-amsa.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-       "ent/iso-amsb.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-       "ent/iso-amsc.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-       "ent/iso-amsn.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-       "ent/iso-amso.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-       "ent/iso-amsr.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-       "ent/iso-box.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-       "ent/iso-cyr1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-       "ent/iso-cyr2.ent"
-
   -- End of catalog data for DocBook XML V4.2 ............................. --
   -- ...................................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.3/catalog.xml docbook-xml-4.5/docbook-4.3/catalog.xml
--- docbook-xml-4.5~/docbook-4.3/catalog.xml	2004-03-31 13:22:12.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/catalog.xml	2007-03-25 23:42:57.000000000 +0200
@@ -49,66 +49,6 @@
 <public publicId="-//OASIS//ENTITIES DocBook Character Entities V4.3//EN"
         uri="dbcentx.mod"/>
 
-<!-- ...................................................................... -->
-<!-- ISO entity sets ...................................................... -->
-
-<public publicId="ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-        uri="ent/iso-dia.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-        uri="ent/iso-num.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Publishing//EN//XML"
-        uri="ent/iso-pub.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES General Technical//EN//XML"
-        uri="ent/iso-tech.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-        uri="ent/iso-lat1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-        uri="ent/iso-lat2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-        uri="ent/iso-grk1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-        uri="ent/iso-grk2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-        uri="ent/iso-grk3.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-        uri="ent/iso-grk4.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-        uri="ent/iso-amsa.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-        uri="ent/iso-amsb.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-        uri="ent/iso-amsc.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-        uri="ent/iso-amsn.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-        uri="ent/iso-amso.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-        uri="ent/iso-amsr.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-        uri="ent/iso-box.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-        uri="ent/iso-cyr1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-        uri="ent/iso-cyr2.ent"/>
-
 <!-- End of catalog data for DocBook XML V4.3 ............................. -->
 <!-- ...................................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.3/dbcentx.mod docbook-xml-4.5/docbook-4.3/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.3/dbcentx.mod	2004-03-31 13:20:20.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -173,133 +173,133 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/iso-amsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 <!--end of ISOamsa.module-->]]>
 
 <!ENTITY % ISOamsb.module "INCLUDE">
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/iso-amsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 <!--end of ISOamsb.module-->]]>
 
 <!ENTITY % ISOamsc.module "INCLUDE">
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/iso-amsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 <!--end of ISOamsc.module-->]]>
 
 <!ENTITY % ISOamsn.module "INCLUDE">
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/iso-amsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 <!--end of ISOamsn.module-->]]>
 
 <!ENTITY % ISOamso.module "INCLUDE">
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/iso-amso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 <!--end of ISOamso.module-->]]>
 
 <!ENTITY % ISOamsr.module "INCLUDE">
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/iso-amsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 <!--end of ISOamsr.module-->]]>
 
 <!ENTITY % ISObox.module "INCLUDE">
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/iso-box.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 <!--end of ISObox.module-->]]>
 
 <!ENTITY % ISOcyr1.module "INCLUDE">
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/iso-cyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 <!--end of ISOcyr1.module-->]]>
 
 <!ENTITY % ISOcyr2.module "INCLUDE">
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/iso-cyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 <!--end of ISOcyr2.module-->]]>
 
 <!ENTITY % ISOdia.module "INCLUDE">
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/iso-dia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 <!--end of ISOdia.module-->]]>
 
 <!ENTITY % ISOgrk1.module "INCLUDE">
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/iso-grk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 <!--end of ISOgrk1.module-->]]>
 
 <!ENTITY % ISOgrk2.module "INCLUDE">
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/iso-grk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 <!--end of ISOgrk2.module-->]]>
 
 <!ENTITY % ISOgrk3.module "INCLUDE">
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/iso-grk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 <!--end of ISOgrk3.module-->]]>
 
 <!ENTITY % ISOgrk4.module "INCLUDE">
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/iso-grk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 <!--end of ISOgrk4.module-->]]>
 
 <!ENTITY % ISOlat1.module "INCLUDE">
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/iso-lat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 <!--end of ISOlat1.module-->]]>
 
 <!ENTITY % ISOlat2.module "INCLUDE">
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/iso-lat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 <!--end of ISOlat2.module-->]]>
 
 <!ENTITY % ISOnum.module "INCLUDE">
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/iso-num.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 <!--end of ISOnum.module-->]]>
 
 <!ENTITY % ISOpub.module "INCLUDE">
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/iso-pub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 <!--end of ISOpub.module-->]]>
 
 <!ENTITY % ISOtech.module "INCLUDE">
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/iso-tech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 <!--end of ISOtech.module-->]]>
 
 <!--end of xml.features-->]]>
diff -urNad docbook-xml-4.5~/docbook-4.3/docbook.cat docbook-xml-4.5/docbook-4.3/docbook.cat
--- docbook-xml-4.5~/docbook-4.3/docbook.cat	2004-03-31 13:22:18.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/docbook.cat	2007-03-25 23:42:57.000000000 +0200
@@ -46,65 +46,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.3//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-       "ent/iso-dia.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-       "ent/iso-num.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-       "ent/iso-pub.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-       "ent/iso-tech.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-       "ent/iso-lat1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-       "ent/iso-lat2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-       "ent/iso-grk1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-       "ent/iso-grk2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-       "ent/iso-grk3.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-       "ent/iso-grk4.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-       "ent/iso-amsa.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-       "ent/iso-amsb.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-       "ent/iso-amsc.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-       "ent/iso-amsn.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-       "ent/iso-amso.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-       "ent/iso-amsr.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-       "ent/iso-box.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-       "ent/iso-cyr1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-       "ent/iso-cyr2.ent"
-
   -- End of catalog data for DocBook XML V4.3 ............................. --
   -- ...................................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.4/catalog.xml docbook-xml-4.5/docbook-4.4/catalog.xml
--- docbook-xml-4.5~/docbook-4.4/catalog.xml	2005-01-27 14:57:19.000000000 +0100
+++ docbook-xml-4.5/docbook-4.4/catalog.xml	2007-03-25 23:42:57.000000000 +0200
@@ -58,66 +58,6 @@
 <public publicId="-//OASIS//ENTITIES DocBook Character Entities V4.4//EN"
         uri="dbcentx.mod"/>
 
-<!-- ...................................................................... -->
-<!-- ISO entity sets ...................................................... -->
-
-<public publicId="ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-        uri="ent/isodia.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-        uri="ent/isonum.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Publishing//EN//XML"
-        uri="ent/isopub.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES General Technical//EN//XML"
-        uri="ent/isotech.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-        uri="ent/isolat1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-        uri="ent/isolat2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-        uri="ent/isogrk1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-        uri="ent/isogrk2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-        uri="ent/isogrk3.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-        uri="ent/isogrk4.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-        uri="ent/isoamsa.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-        uri="ent/isoamsb.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-        uri="ent/isoamsc.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-        uri="ent/isoamsn.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-        uri="ent/isoamso.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-        uri="ent/isoamsr.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-        uri="ent/isobox.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-        uri="ent/isocyr1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-        uri="ent/isocyr2.ent"/>
-
 <!-- End of catalog data for DocBook XML V4.4 ............................. -->
 <!-- ...................................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.4/dbcentx.mod docbook-xml-4.5/docbook-4.4/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.4/dbcentx.mod	2005-01-27 14:57:19.000000000 +0100
+++ docbook-xml-4.5/docbook-4.4/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -173,133 +173,133 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/isoamsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 <!--end of ISOamsa.module-->]]>
 
 <!ENTITY % ISOamsb.module "INCLUDE">
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/isoamsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 <!--end of ISOamsb.module-->]]>
 
 <!ENTITY % ISOamsc.module "INCLUDE">
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/isoamsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 <!--end of ISOamsc.module-->]]>
 
 <!ENTITY % ISOamsn.module "INCLUDE">
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/isoamsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 <!--end of ISOamsn.module-->]]>
 
 <!ENTITY % ISOamso.module "INCLUDE">
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/isoamso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 <!--end of ISOamso.module-->]]>
 
 <!ENTITY % ISOamsr.module "INCLUDE">
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/isoamsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 <!--end of ISOamsr.module-->]]>
 
 <!ENTITY % ISObox.module "INCLUDE">
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/isobox.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 <!--end of ISObox.module-->]]>
 
 <!ENTITY % ISOcyr1.module "INCLUDE">
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/isocyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 <!--end of ISOcyr1.module-->]]>
 
 <!ENTITY % ISOcyr2.module "INCLUDE">
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/isocyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 <!--end of ISOcyr2.module-->]]>
 
 <!ENTITY % ISOdia.module "INCLUDE">
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/isodia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 <!--end of ISOdia.module-->]]>
 
 <!ENTITY % ISOgrk1.module "INCLUDE">
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/isogrk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 <!--end of ISOgrk1.module-->]]>
 
 <!ENTITY % ISOgrk2.module "INCLUDE">
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/isogrk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 <!--end of ISOgrk2.module-->]]>
 
 <!ENTITY % ISOgrk3.module "INCLUDE">
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/isogrk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 <!--end of ISOgrk3.module-->]]>
 
 <!ENTITY % ISOgrk4.module "INCLUDE">
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/isogrk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 <!--end of ISOgrk4.module-->]]>
 
 <!ENTITY % ISOlat1.module "INCLUDE">
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/isolat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 <!--end of ISOlat1.module-->]]>
 
 <!ENTITY % ISOlat2.module "INCLUDE">
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/isolat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 <!--end of ISOlat2.module-->]]>
 
 <!ENTITY % ISOnum.module "INCLUDE">
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/isonum.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 <!--end of ISOnum.module-->]]>
 
 <!ENTITY % ISOpub.module "INCLUDE">
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/isopub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 <!--end of ISOpub.module-->]]>
 
 <!ENTITY % ISOtech.module "INCLUDE">
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/isotech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 <!--end of ISOtech.module-->]]>
 
 <!--end of xml.features-->]]>
diff -urNad docbook-xml-4.5~/docbook-4.4/docbook.cat docbook-xml-4.5/docbook-4.4/docbook.cat
--- docbook-xml-4.5~/docbook-4.4/docbook.cat	2005-01-27 14:57:19.000000000 +0100
+++ docbook-xml-4.5/docbook-4.4/docbook.cat	2007-03-25 23:42:57.000000000 +0200
@@ -49,65 +49,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.4//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-       "ent/isodia.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-       "ent/isonum.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-       "ent/isopub.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-       "ent/isotech.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-       "ent/isolat1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-       "ent/isolat2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-       "ent/isogrk1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-       "ent/isogrk2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-       "ent/isogrk3.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-       "ent/isogrk4.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-       "ent/isoamsa.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-       "ent/isoamsb.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-       "ent/isoamsc.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-       "ent/isoamsn.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-       "ent/isoamso.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-       "ent/isoamsr.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-       "ent/isobox.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-       "ent/isocyr1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-       "ent/isocyr2.ent"
-
   -- End of catalog data for DocBook XML V4.4 ............................. --
   -- ...................................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.5/catalog.xml docbook-xml-4.5/docbook-4.5/catalog.xml
--- docbook-xml-4.5~/docbook-4.5/catalog.xml	2006-10-03 15:26:00.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/catalog.xml	2007-03-25 23:42:57.000000000 +0200
@@ -58,66 +58,6 @@
 <public publicId="-//OASIS//ENTITIES DocBook Character Entities V4.5//EN"
         uri="dbcentx.mod"/>
 
-<!-- ...................................................................... -->
-<!-- ISO entity sets ...................................................... -->
-
-<public publicId="ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-        uri="ent/isodia.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-        uri="ent/isonum.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Publishing//EN//XML"
-        uri="ent/isopub.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES General Technical//EN//XML"
-        uri="ent/isotech.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-        uri="ent/isolat1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-        uri="ent/isolat2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-        uri="ent/isogrk1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-        uri="ent/isogrk2.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-        uri="ent/isogrk3.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-        uri="ent/isogrk4.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-        uri="ent/isoamsa.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-        uri="ent/isoamsb.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-        uri="ent/isoamsc.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-        uri="ent/isoamsn.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-        uri="ent/isoamso.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-        uri="ent/isoamsr.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-        uri="ent/isobox.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-        uri="ent/isocyr1.ent"/>
-
-<public publicId="ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-        uri="ent/isocyr2.ent"/>
-
 <!-- End of catalog data for DocBook XML V4.5 ............................. -->
 <!-- ...................................................................... -->
 
diff -urNad docbook-xml-4.5~/docbook-4.5/dbcentx.mod docbook-xml-4.5/docbook-4.5/dbcentx.mod
--- docbook-xml-4.5~/docbook-4.5/dbcentx.mod	2006-10-03 15:26:00.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/dbcentx.mod	2007-03-25 23:42:57.000000000 +0200
@@ -173,133 +173,133 @@
 <![%ISOamsa.module;[
 <!ENTITY % ISOamsa PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-"ent/isoamsa.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsa.ent">
 <!--end of ISOamsa.module-->]]>
 
 <!ENTITY % ISOamsb.module "INCLUDE">
 <![%ISOamsb.module;[
 <!ENTITY % ISOamsb PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-"ent/isoamsb.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsb.ent">
 <!--end of ISOamsb.module-->]]>
 
 <!ENTITY % ISOamsc.module "INCLUDE">
 <![%ISOamsc.module;[
 <!ENTITY % ISOamsc PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-"ent/isoamsc.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsc.ent">
 <!--end of ISOamsc.module-->]]>
 
 <!ENTITY % ISOamsn.module "INCLUDE">
 <![%ISOamsn.module;[
 <!ENTITY % ISOamsn PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-"ent/isoamsn.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsn.ent">
 <!--end of ISOamsn.module-->]]>
 
 <!ENTITY % ISOamso.module "INCLUDE">
 <![%ISOamso.module;[
 <!ENTITY % ISOamso PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-"ent/isoamso.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamso.ent">
 <!--end of ISOamso.module-->]]>
 
 <!ENTITY % ISOamsr.module "INCLUDE">
 <![%ISOamsr.module;[
 <!ENTITY % ISOamsr PUBLIC
 "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-"ent/isoamsr.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOamsr.ent">
 <!--end of ISOamsr.module-->]]>
 
 <!ENTITY % ISObox.module "INCLUDE">
 <![%ISObox.module;[
 <!ENTITY % ISObox PUBLIC
 "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-"ent/isobox.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISObox.ent">
 <!--end of ISObox.module-->]]>
 
 <!ENTITY % ISOcyr1.module "INCLUDE">
 <![%ISOcyr1.module;[
 <!ENTITY % ISOcyr1 PUBLIC
 "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-"ent/isocyr1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr1.ent">
 <!--end of ISOcyr1.module-->]]>
 
 <!ENTITY % ISOcyr2.module "INCLUDE">
 <![%ISOcyr2.module;[
 <!ENTITY % ISOcyr2 PUBLIC
 "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-"ent/isocyr2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOcyr2.ent">
 <!--end of ISOcyr2.module-->]]>
 
 <!ENTITY % ISOdia.module "INCLUDE">
 <![%ISOdia.module;[
 <!ENTITY % ISOdia PUBLIC
 "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-"ent/isodia.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOdia.ent">
 <!--end of ISOdia.module-->]]>
 
 <!ENTITY % ISOgrk1.module "INCLUDE">
 <![%ISOgrk1.module;[
 <!ENTITY % ISOgrk1 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-"ent/isogrk1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk1.ent">
 <!--end of ISOgrk1.module-->]]>
 
 <!ENTITY % ISOgrk2.module "INCLUDE">
 <![%ISOgrk2.module;[
 <!ENTITY % ISOgrk2 PUBLIC
 "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-"ent/isogrk2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk2.ent">
 <!--end of ISOgrk2.module-->]]>
 
 <!ENTITY % ISOgrk3.module "INCLUDE">
 <![%ISOgrk3.module;[
 <!ENTITY % ISOgrk3 PUBLIC
 "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-"ent/isogrk3.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk3.ent">
 <!--end of ISOgrk3.module-->]]>
 
 <!ENTITY % ISOgrk4.module "INCLUDE">
 <![%ISOgrk4.module;[
 <!ENTITY % ISOgrk4 PUBLIC
 "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-"ent/isogrk4.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOgrk4.ent">
 <!--end of ISOgrk4.module-->]]>
 
 <!ENTITY % ISOlat1.module "INCLUDE">
 <![%ISOlat1.module;[
 <!ENTITY % ISOlat1 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-"ent/isolat1.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat1.ent">
 <!--end of ISOlat1.module-->]]>
 
 <!ENTITY % ISOlat2.module "INCLUDE">
 <![%ISOlat2.module;[
 <!ENTITY % ISOlat2 PUBLIC
 "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-"ent/isolat2.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOlat2.ent">
 <!--end of ISOlat2.module-->]]>
 
 <!ENTITY % ISOnum.module "INCLUDE">
 <![%ISOnum.module;[
 <!ENTITY % ISOnum PUBLIC
 "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-"ent/isonum.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOnum.ent">
 <!--end of ISOnum.module-->]]>
 
 <!ENTITY % ISOpub.module "INCLUDE">
 <![%ISOpub.module;[
 <!ENTITY % ISOpub PUBLIC
 "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-"ent/isopub.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOpub.ent">
 <!--end of ISOpub.module-->]]>
 
 <!ENTITY % ISOtech.module "INCLUDE">
 <![%ISOtech.module;[
 <!ENTITY % ISOtech PUBLIC
 "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-"ent/isotech.ent">
+"/usr/share/xml/entities/xml-iso-entities-8879.1986/ISOtech.ent">
 <!--end of ISOtech.module-->]]>
 
 <!--end of xml.features-->]]>
diff -urNad docbook-xml-4.5~/docbook-4.5/docbook.cat docbook-xml-4.5/docbook-4.5/docbook.cat
--- docbook-xml-4.5~/docbook-4.5/docbook.cat	2006-10-03 15:26:00.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/docbook.cat	2007-03-25 23:42:57.000000000 +0200
@@ -49,65 +49,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN//XML"
-       "ent/isodia.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN//XML"
-       "ent/isonum.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN//XML"
-       "ent/isopub.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN//XML"
-       "ent/isotech.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN//XML"
-       "ent/isolat1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN//XML"
-       "ent/isolat2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN//XML"
-       "ent/isogrk1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN//XML"
-       "ent/isogrk2.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN//XML"
-       "ent/isogrk3.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN//XML"
-       "ent/isogrk4.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN//XML"
-       "ent/isoamsa.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN//XML"
-       "ent/isoamsb.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN//XML"
-       "ent/isoamsc.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN//XML"
-       "ent/isoamsn.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN//XML"
-       "ent/isoamso.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN//XML"
-       "ent/isoamsr.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN//XML"
-       "ent/isobox.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN//XML"
-       "ent/isocyr1.ent"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN//XML"
-       "ent/isocyr2.ent"
-
   -- End of catalog data for DocBook XML V4.5 ............................. --
   -- ...................................................................... --
